pulsar-broker/src/test/java/org/apache/pulsar/broker/transaction/TransactionTest.java

```diff
@@ -53,23 +61,26 @@
 
     private static final String TENANT = "tnx";
     private static final String NAMESPACE1 = TENANT + "/ns1";
+    private static final int NUM_BROKER = 1;
+    private static final int NUM_TC_PER = 1;
```

Comment:
what about NUM_PARTITIONS ?

Reply:
Thank you for your help and guidance, 
your meticulous spirit and outstanding design concept have benefited me a lot